src/main/java/ch/njol/skript/conditions/CondIsDivisibleBy.java

```diff
@@ -1,19 +1,26 @@
 package ch.njol.skript.conditions;
 
 import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Condition;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 import org.bukkit.event.Event;
 import org.jetbrains.annotations.Nullable;
 
+@Name("Divisible By")
+@Description("Check if a number is divisible by another number.")
+@Examples("5 is divisible by 5")
```

Comment:
```suggestion
@Examples("if 5 is divisible by 5:")
```
id rather there be more examples, show off the other syntax too